app/controllers/admin/cases_controller.rb

```diff
@@ -12,7 +12,7 @@ class Admin::CasesController < ApplicationController
                 ]
 
   def create
-    @correspondence_type_abbreviation = params.fetch(:correspondence_type).downcase
+    @correspondence_type_key = params.fetch(:correspondence_type).downcase
```

Comment:
Just curious why correspondence_type_abbreviation was changed?

Reply:
Because we have a real abbreviation used that is in all-caps (FOI), so I wanted it to be clearer that this is the lower-cased version.